app/src/renderer/components/governance/TabParameters.vue

```diff
@@ -0,0 +1,279 @@
+                    ? `${parseFloat(
+                        governanceParameters.parameters.tallying.threshold
+                      ) * 100} %`
+                    : `n/a`
+                }}
+              </dd>
+            </dl>
+            <dl class="info_dl">
+              <dt>
+                Veto
+                <i
+                  v-tooltip.top="tallyingTooltips.veto"
+                  class="material-icons info-button"
+                  >info_outline</i
+                >
+              </dt>
+              <dd>
+                {{
+                  governanceParameters.parameters.tallying.veto
+                    ? `${parseFloat(
+                        governanceParameters.parameters.tallying.veto
+                      ) * 100} %`
+                    : `n/a`
+                }}
+              </dd>
+            </dl>
+          </div>
+          <div class="column">
+            <dl class="info_dl">
+              <dt>
+                Governance Penalty
+                <i
+                  v-tooltip.top="tallyingTooltips.governance_penalty"
+                  class="material-icons info-button"
+                  >info_outline</i
+                >
+              </dt>
+              <dd>
+                {{
+                  governanceParameters.parameters.tallying.governance_penalty
+                    ? governanceParameters.parameters.tallying
+                        .governance_penalty
+                    : `n/a`
+                }}
+              </dd>
+            </dl>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div>
+      <h3>
+        Voting Parameters
+        <i
+          v-tooltip.top="votingTooltips.description"
+          class="material-icons info-button"
+          >info_outline</i
+        >
+      </h3>
+      <div class="parameters__details parameters__section">
+        <div class="row">
+          <div class="column">
+            <dl class="info_dl">
+              <dt>
+                Voting Period
+                <i
+                  v-tooltip.top="votingTooltips.voting_period"
+                  class="material-icons info-button"
+                  >info_outline</i
+                >
+              </dt>
+              <dd>
+                {{
+                  governanceParameters.parameters.voting.voting_period
+                    ? votingPeriodInDays + ` days`
+                    : `n/a`
+                }}
+              </dd>
+            </dl>
+          </div>
+          <div class="column"></div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapGetters } from "vuex"
+import { TmBtn, TmListItem, TmPage, TmPart, TmToolBar } from "@tendermint/ui"
+export default {
+  name: `tab-staking-parameters`,
+  components: {
+    TmBtn,
+    TmListItem,
+    TmPage,
+    TmPart,
+    TmToolBar
+  },
+  data: () => ({
+    depositTooltips: {
+      description: `Parameters around proposals' deposits on governance`,
```

Comment:
```suggestion
      description: `Governance parameters for the Cosmos Hub`,
```

Reply:
this is the description for the governance deposits params only